src/components/CommentTeaser/CommentTeaser.js

```diff
@@ -202,7 +210,10 @@ export const CommentTeaser = ({
                       {inQuotes(discussion.title)}
                       {newPage && (
                         <span {...styles.icon}>
-                          <NewPage size={ICON_SIZE} fill={colors.disabled} />
+                          <NewPage
+                            size={ICON_SIZE}
+                            fill={colorScheme.getCSSColor('disabled')}
```

Comment:
```suggestion
                            {...colorScheme.set('fill', 'disabled')}
```

But actually we don't use the whole `newPage` prop anymore. If you feel like it: rm it including from docs (`CommentTeaser/docs.md`)